content/en/apps/guides/data/hydration.md

```diff
@@ -0,0 +1,185 @@
+    }
+  },
+  "contact": {
+    "_id": "contact_id",
+    "name": "Primary contact",
+    "phone": "555 111 222"
+  },
+  "linked_docs": {
+    "tag1": {
+      "_id": "sibling_id",
+      "name": "Sibling clinic",
+      "type": "clinic"
+    },
+    "tag2": {
+      "_id": "supervisor_id",
+      "name": "The supervisor",
+      "type": "person"
+    }
+  }
+}
+``` 
+when minified, becomes:
+```json
+{
+  "_id": "clinic_uuid",
+  "name": "Clinic",
+  "type": "clinic",
+  "parent": {
+    "_id": "health_center_id",
+    "parent": {
+      "_id": "district_hospital_id"
+    }
+  },
+  "contact": {
+    "_id": "contact_id"
+  },
+  "linked_contacts": {
+    "tag1": "sibling_id",
+    "tag2": "supervisor_id"
+  }
+}
+```
+
+The following properties are minified:
+1. `parent` and recursively, every ancestor
+1. `contact`
+1. `patient` and `place` are removed from reports
+1. *As of 3.10* `linked_docs` when minifying a contact
+
+### Hydration
+
+Hydration represents the inverse process to minification, where a stored id representing a connected document is replaced with the corresponding document's content. 
+  
+Minified doc: 
+```json
+{
+  "_id": "clinic_uuid",
+  "name": "Clinic",
+  "type": "clinic",
+  "parent": {
+    "_id": "health_center_id",
+    "parent": {
+      "_id": "district_hospital_id"
+    }
+  },
+  "contact": {
+    "_id": "contact_id"
+  },
+  "linked_contacts": {
+    "tag1": "sibling_id",
+    "tag2": "supervisor_id"
+  }
+}
+```
+when hydrated becomes:
+```json
+{
+  "_id": "clinic_uuid",
+  "name": "Clinic",
+  "type": "clinic",
+  "parent": {
+    "_id": "health_center_id",
+    "name": "Health Center",
+    "type": "health_center",
+    "contact": {
+      "_id": "supervisor_id",
+      "name": "Supervisor",
+      "type": "person",
+      "parent": { 
+        "_id": "parent_id" 
+      }
+    },
+    "parent": {
+      "_id": "district_hospital_id",
+      "name" : "District",
+      "type": "district_hospital",
+      "contact": {
+        "_id": "manager_id",
+        "name": "Manager",
+        "type": "person",
+        "parent": {
+          "_id": "parent_id"
+        }       
+      }
+    }
+  },
+  "contact": {
+    "_id": "contact_id",
+    "name": "Primary contact",
+    "phone": "555 111 222",
+    "parent": {
+      "_id": "parent_id"
+    } 
+  },
+  "linked_docs": {
+    "tag1": {
+      "_id": "sibling_id",
+      "name": "Sibling clinic",
+      "type": "clinic"
+    },
+    "tag2": {
+      "_id": "supervisor_id",
+      "name": "The supervisor",
+      "type": "person"
+    }
+  }
+}
+``` 
+
+There are two types of hydration:
+* *shallow* hydration - when the id is replaced with the document's content
+* *deep* hydration - when the id is replaced with the document's content and, recursively, deeply hydrate every ancestor 
+
+###### A hydrated contact has
+
+- deeply hydrated `parent` along with every ancestor
+- shallowly hydrated primary `contact`
+- *as of 3.10* shallowly hydrated `linked_docs`
```

Comment:
I like the version number at the end, because it's easier to scan down the list and find the thing you're interesting in.

```suggestion
- shallowly hydrated `linked_docs` (*as of 3.10*)
```